WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -176,7 +177,8 @@ private async Task<ImmutableArray<AliceClient>> CreateRegisterAndConfirmCoinsAsy
 				try
 				{
 					// Alice client requests are inherently linkable to each other, so the circuit can be reused
-					var arenaRequestHandler = new WabiSabiHttpApiClient(HttpClientFactory.NewBackendHttpClient(Mode.SingleCircuitPerLifetime));
+					PersonCircuit personCircuit = new();
+					var arenaRequestHandler = new WabiSabiHttpApiClient(HttpClientFactory.NewBackendHttpClient(Mode.SingleCircuitPerLifetime, personCircuit));
```

Comment:
`NewBackendHttpClient` is hard to use, I would add methods to make the usage obvious.

Reply:
Yeah, I had that method already added but changed my mind. I'll add it again.